extensions/scarb-cairo-run/src/main.rs

```diff
@@ -61,13 +62,13 @@ fn main() -> Result<()> {
 
     ui.print(Status::new("Running", &package.name));
 
-    let sierra_program = cairo_lang_sierra::ProgramParser::new()
-        .parse(
-            &fs::read_to_string(path.clone())
-                .with_context(|| format!("failed to read Sierra file: {path}"))?,
-        )
-        .map_err(|e| anyhow!("{e}"))
-        .with_context(|| format!("failed to parse sierra program: {path}"))?;
+    let sierra_program = serde_json::from_str::<VersionedProgram>(
+        &fs::read_to_string(path.clone())
+            .with_context(|| format!("failed to read Sierra file: {path}"))?,
+    )
+    .map_err(|e| anyhow!("{e}"))
```

Comment:
Do we need this line? It was added because of the way that lalrpop has constructed error structs.

Reply:
```suggestion
```